script/dbdefs_to_js.pl

```diff
@@ -132,21 +132,32 @@ sub get_value {
 my $server_code = "// \@flow strict\n";
 my $client_code = "// \@flow strict\n";
 
+my (@all_client_defs, @all_server_defs);
+
 for my $conversion (@conversions) {
     my ($defs, $convert, $flowtype) = @{$conversion}{qw(defs convert flowtype)};
 
     for my $def (@$defs) {
         my @raw_value = get_value($def);
         my $json_value = $json->encode(${$convert->(@raw_value)});
-        my $line = "exports.$def = ($json_value/*: $flowtype */);\n";
+        my $line = "export const $def/*: $flowtype */ = $json_value;\n";
         $server_code .= $line;
-        $client_code .= $line if $CLIENT_DEFS{$def};
+        push @all_server_defs, $def;
+        if ($CLIENT_DEFS{$def}) {
+            $client_code .= $line;
+            push @all_client_defs, $def;
+        }
     }
 }
 
+# Continue to allow importing as `import DBDefs from ...` by providing a
+# default export.
```

Comment:
I preferred to avoid having to fix any existing imports.  I had to rebase this branch at least 50x, and since I was changing all of the DBDefs imports in other commits, this would've caused additional conflicts.